src/bots/filler.ts

```diff
@@ -2269,6 +2317,74 @@ export class FillerBot extends TxThreaded implements Bot {
 		return true;
 	}
 
+	public findTriggerableNodesWithMakers(
+		dlob: DLOB,
+		marketIndex: number,
+		slot: number,
+		oraclePriceData: OraclePriceData,
+		marketType: MarketType,
+		stateAccount: StateAccount
+	): NodeToTriggerWithMakers[] {
+		const baseTriggerable = dlob.findNodesToTrigger(
+			marketIndex,
+			slot,
+			oraclePriceData.price,
+			marketType,
+			stateAccount
+		);
+
+		const triggerWithMaker: NodeToTriggerWithMakers[] = [];
+
+		for (const nodeObj of baseTriggerable) {
+			const order = nodeObj.node.order;
+			let isFillable = false;
+
+			if (
+				isVariant(order.orderType, 'market') ||
+				isVariant(order.orderType, 'triggerMarket')
+			) {
+				isFillable = true;
+			} else if (isVariant(order.orderType, 'triggerLimit')) {
+				if (
+					isVariant(order.triggerCondition, 'triggeredAbove') ||
+					isVariant(order.triggerCondition, 'above')
+				) {
+					isFillable = order.price.lte(oraclePriceData.price);
+				} else if (
+					isVariant(order.triggerCondition, 'triggeredBelow') ||
+					isVariant(order.triggerCondition, 'below')
+				) {
+					isFillable = order.price.gte(oraclePriceData.price);
+				}
+			}
+
+			const NUM_MAKERS = 4;
+			if (isFillable) {
+				const makers = dlob.getBestMakers({
+					marketIndex,
+					marketType,
+					direction: order.direction,
```

Comment:
i think you need to use the opposite direction, if order is bid you need asks etc.